projectDocs/dev/codingStandards.md

```diff
@@ -92,28 +105,66 @@ self.copySettingsButton = wx.Button(
 ```
 
 ### Imports
+
 * Unused imports should be removed where possible.
-  - Anything imported into a (sub)module can also be imported from that submodule.
-  - As a result, removing unused imports may break compatibility, and should be done in compatibility breaking releases (see `deprecations.md`).
-* Unused imports will give a lint warning. These can be handled the following ways:
-  - If these imports are intended to be imported from other modules, they can be included in a definition for `__all__`. This will override and define the symbols imported when performing a star import, e.g. `from module import *`.
-  - Otherwise, with a comment like `# noqa: <explanation>`.
+  * Anything imported into a (sub)module can also be imported from that submodule.
+  * As a result, removing unused imports may break compatibility, and should be done in compatibility breaking releases (see `deprecations.md`).
+* Unused imports will give a lint warning.
+  These can be handled the following ways:
+  * If these imports are intended to be imported from other modules, they can be included in a definition for `__all__`.
+  This will override and define the symbols imported when performing a star import, e.g. `from module import *`.
+  * Otherwise, with a comment like `# noqa: <explanation>`.
 
 ### Considering future backwards compatibility
 
 When writing new code, consider how the code can be moved in future while retaining backwards compatibility.
 Refer to the [limitations to retaining backwards compatibility](./deprecations.md#limitations-to-retaining-backwards-compatibility).
 
 In summary:
-- Avoid module level global variables.
+
+* Avoid module level global variables.
 Any module level variables should be prefixed with an underscore and be encapsulated, e.g. via getters and setters.
-- Avoid code which executes at import time.
+* Avoid code which executes at import time.
 Instead use initializer functions.
 
 ### Docstrings
 
-Docstrings should use [Sphinx format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html).
-Providing type information in docstrings is discouraged, instead use python's type annotations.
+Docstrings should use [Sphinx format without types](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html), and follow [PEP 257 conventions](https://peps.python.org/pep-0257/).
```

Comment:
I find this cheat sheet helpful as well: https://devguide.python.org/documentation/markup/